diff -up nailer-0.4.3/src/main.c.glib nailer-0.4.3/src/main.c
diff -up nailer-0.4.3/src/main.c.glib nailer-0.4.3/src/main.c
--- nailer-0.4.3/src/main.c.glib	2008-09-02 22:22:54.000000000 +0200
+++ nailer-0.4.3/src/main.c	2012-02-11 15:23:16.914812245 +0100
@@ -24,7 +24,6 @@
 
 #include <glib.h>
 #include <glib/gstdio.h>
-#include <glib/gstrfuncs.h>
 #include <gdk-pixbuf/gdk-pixbuf.h>
 #include <string.h>
 #include "left.h"
